src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/ThemeController.cs

```diff
@@ -28,9 +29,8 @@ public class ThemeController : ControllerBase
         }
 
         [HttpGet("themes")]
-        [BlobCache(typeof(PublicationTreeCacheKey))]
-        public async Task<IList<ThemeTree<PublicationTreeNode>>> GetThemes(
-            [FromQuery(Name = "publicationFilter")] PublicationTreeFilter? filter = null)
+        public async Task<IList<ThemeTree<PublicationTreeNode>>> GetPublicationTree(
+            [Required][FromQuery(Name = "publicationFilter")] PublicationTreeFilter filter)
```

Comment:
So I tried this with an without `[Required]` and it made no difference to the behaviour.

If you hit the endpoint with `GET {content_api_url}/themes?publicationFilter=blah` or just `GET {content_api_url}/themes`, it'll default to `FindStatistics` because it's first in the `PublicationTreeFilter` enum. This wasn't the behaviour I was expecting.

`GET {content_api_url}/themes?publicationFilter=DataTables` behaves as expected.

I'm inclined to leave this - I don't think defaulting to `FindStatistics` is a problem. I've removed the `[Required]` since it doesn't appear to do anything. But happy to do something else if others prefer.

Reply:
After discussing on Slack with Ben, I've made `filter` nullable and return a `BadRequestResult` if it is null.